lib/dpl/providers/bintray.rb

```diff
@@ -112,19 +115,41 @@ def publish_version
         post(path(:version_publish))
       end
 
+      def update_files
+        files.each do |file|
+          info :upload_file, source: file.source, target: file.target
+          update_file(file)
+        end
+      end
+
+      def update_file(file)
+        retrying(max: 10, pause: 5) do
+          put(path(:file_metadata, target: file.target), list_in_downloads: true)
```

Comment:
Why does `list_in_downloads` is always `true`? And I think that the request body should be converted into json.
I was expecting something like this:

```suggestion
          put(path(:file_metadata, target: file.target), {list_in_downloads: file.download}.to_json)
```